frontend/src/reducers/tables.js

```diff
@@ -3,6 +3,7 @@ import { get } from 'lodash';
 import { createSelector } from 'reselect';
 
 import * as types from '../constants/ActionTypes';
+import { UPDATE_TABLE_SELECTION } from '../constants/actions/TableTypes';
 
 export const tableState = {
```

Comment:
@siemiatj  @petrican  IMHO this shall be called initialTableState

Reply:
Ok. Renaming it.